graph/R-GAT/README.md

```diff
@@ -209,6 +209,12 @@ Add the `--accuracy` to the command to run the benchmark
 python3 main.py --dataset igbh --dataset-path igbh/ --accuracy --model-path model/ [--model-path <path_to_ckpt>] [--in-memory] [--device <cpu or gpu>] [--dtype <fp16 or fp32>] [--scenario <SingleStream, MultiStream, Server or Offline>] [--layout <COO, CSC or CSR>]
 ```
 
+**NOTE:** For official submissions you should submit the results of the accuracy run in a file called `accuracy.txt` with the following format:
```

Comment:
Hi @pgmpablo157321 currently the reference accuracy script is doing output in json format. Should we update that here?
https://github.com/mlcommons/inference/blob/master/graph/R-GAT/tools/accuracy_igbh.py#L121

Reply:
Makes sense, I'll change it. BTW, do you know what we usually do to generate the hash? I see the hash in all the previous submissions, for example:

https://github.com/mlcommons/inference_results_v4.1/blob/main/closed/AMD/results/1xMI300X_2xEPYC-9374F/llama2-70b-99.9/Offline/accuracy/accuracy.txt

But don't see the code to generate it

Reply:
Thank you @pgmpablo157321 The hash value is written by the truncate_accuracy script. 

https://github.com/mlcommons/inference/blob/master/tools/submission/truncate_accuracy_log.py#L235